README.md

```diff
@@ -30,25 +54,225 @@ npm install --save ipfsd-ctl
 
 IPFS daemons are already easy to start and stop, but this module is here to do it from JavaScript itself.
 
+### Local node
+
 ```js
 // Start a disposable node, and get access to the api
-// print the node id, and kill the temporary daemon
+// print the node id, and stop the temporary daemon
 
 // IPFS_PATH will point to /tmp/ipfs_***** and will be
 // cleaned up when the process exits.
 
-var ipfsd = require('ipfsd-ctl')
+const daemonFactory = require('ipfsd-ctl')
+const local = daemonFactory.localController
 
-ipfsd.disposableApi(function (err, ipfs) {
-  ipfs.id(function (err, id) {
+local.spawn(function (err, ipfsd) {
```

Comment:
What are the options to select:

- spawning a disposable node
- spawning a node using the ~/.ipfs repo
- spawning a node using the ipfs binary available?